### PR TITLE
[Cosmos] Live test fixes

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cosmos/azure-cosmos/test/crud_tests.py [hunk 1]
```diff
@@ -1977,6 +1985,11 @@ def _test_create_indexing_policy_with_composite_and_spatial_indexes(self, is_nam
         container_definition = {'id': container_id, 'indexingPolicy': indexing_policy}
         created_container = self.client.CreateContainer(self.GetDatabaseLink(db, is_name_based), container_definition)
         read_indexing_policy = created_container['indexingPolicy']
+
+        # All types are returned for spatial Indexes
+        indexing_policy['spatialIndexes'][0]['types'].append('MultiPolygon')
```
Comment (srinathnarayanan) [inline sdk/cosmos/azure-cosmos/test/crud_tests.py]: we need to remove the 2 disabled tests

Comment (annatisch) [reply sdk/cosmos/azure-cosmos/test/crud_tests.py]: Done :)

**Pull request merged**